notebooks/WIBO-critical-path/Multiple activities/objects/activities/day_work activity_001 wid1 A 1713830400.0.json

```diff
@@ -0,0 +1 @@
+{"name": "day_work activity_001 wid1 A 1713830400.0", "ID": "3e4264ac-819a-4071-a098-4ee398296ca5", "log": {"Timestamp": [1713858000.0, 1713873600.0], "ActivityID": ["3e4264ac-819a-4071-a098-4ee398296ca5", "3e4264ac-819a-4071-a098-4ee398296ca5"], "ActivityState": ["START", "STOP"], "ObjectState": [{}, {}], "ActivityLabel": [{}, {}]}}
```

Comment:
can you remove this use case with all its json files and notebooks. We never really used this example and it does not belong it the TU master